src/utils/builder/edit_member.rs

```diff
@@ -2,27 +2,33 @@ use serde_json::builder::ObjectBuilder;
 use std::default::Default;
 use ::model::{ChannelId, RoleId};
 
+/// A builder that edits properties of a ['Member'] via ['Context::edit_member'].
+
 pub struct EditMember(pub ObjectBuilder);
 
 impl EditMember {
+    /// Deafens a member.
     pub fn deafen(self, deafen: bool) -> Self {
         EditMember(self.0.insert("deaf", deafen))
     }
 
+    /// Mutes a member.
     pub fn mute(self, mute: bool) -> Self {
         EditMember(self.0.insert("mute", mute))
     }
 
+    /// Changes a members nickname.
```

Comment:
This should mention that passing an empty string will reset the nickname. Maybe this type should be changed to an Option, but for now that's what it is.